src/actions/dataset.ts

```diff
@@ -63,6 +61,8 @@ export function datasetLoad(name: string, dataset: Data): DatasetLoad {
         .then(response => response.json()) // TODO: handle error
         .then(data => fetchCompassQLBuildSchema(data, config)) // TODO: handle error
         .then(schema => {
+          // Clear filters
+          dispatch({ type: FILTER_CLEAR});
```

Comment:
I'm not sure about that but they should be consistent.  Maybe move shelf clear to before for consistency?